app/services/password_resets/create_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module PasswordResets
+  class CreateService < BaseService
+    def initialize(user:)
+      @user = user
+
+      super
+    end
+
+    def call
+      return result.not_found_failure!(resource: 'user') if user.blank?
+
+      password_reset = PasswordReset.create!(
+        user_id: user.id,
+        token: SecureRandom.hex(20),
+        expire_at: DateTime.now + 30.minutes,
```

Comment:
I guess you should use `Time.current + 30.minutes`

Reply:
Yeah, read some article in the meantime about why :)

Changed it everywhere it was used in this PR